subprojects/gradle/cd/src/main/kotlin/com/avito/ci/internal/ReportKey.kt

```diff
@@ -0,0 +1,21 @@
+package com.avito.ci.internal
+
+import com.avito.report.model.ReportCoordinates
+import java.util.Locale
+
+/**
+ * RunId doesn't matter here, because it's stable for a single build
+ */
+internal data class ReportKey(val planSlug: String, val jobSlug: String) {
+
+    fun appendToTaskName(taskName: String): String {
```

Comment:
So, I don't think that this class should contains this function

Reply:
We have an abstraction leak here

Reply:
why so? it's an internal data class, only to make a clear intention

Reply:
I will try to share my opinion.
What I see, we have a model `ReportKey` it's a data for report-viewer
Should the report-viewer know about the Gradle abstraction as a task? I don't think so
But Gradle task or Gradle plugin could know about the ReportKey and how modify it to a `task` name

Reply:
i don't see it,
but i'll just copy paste a function

Reply:
I don't have an idea to control everything in our codebase but If you asked me I would move this function to the plugin